include/client/tickets.inc.php

```diff
@@ -201,19 +201,19 @@
     <thead>
         <tr>
             <th nowrap>
-                <a href="tickets.php?sort=ID&order=<?php echo $negorder; ?><?php echo $qstr; ?>" title="Sort By Ticket ID"><?php echo __('Ticket #');?></a>
+                <a href="tickets.php?sort=ID&order=<?php echo $negorder; ?><?php echo $qstr; ?>" title="<?php echo sprintf(__('Sort by %s %s'), __('Ticket #'), __($negorder)); ?>"><?php echo __('Ticket #');?></a>
```

Comment:
"Sort By Ticket # DESC" isn't too friendly... how about "Sort By Ticket # in Descending Order"